Alloy/template/app.js

```diff
@@ -7,5 +7,10 @@ var Alloy = require('alloy'),
 	_ = Alloy._,
 	Backbone = Alloy.Backbone;
 
+(function(exports) {
```

Comment:
Not sure if this should be named `exports`. We've seen confusion in Alloy controllers where `exports` really is the controller's `this` and not the CommonJS exports people take it for.

Maybe `globals`